src/components/navigation/questionNavigation/QuestionStepper.tsx

```diff
@@ -70,6 +70,15 @@ export const QuestionStepper = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currentQuestion, appData]);
 
+  // Because default current idx is set to -1
+  // to active selection of create question button,
+  // it has to be set to 0.
+  useEffect(() => {
+    if (currentIdx === -1) {
+      setCurrentIdx(0);
+    }
+  }, [currentIdx, setCurrentIdx]);
+
```

Comment:
I don't get it, why not have currentIdx to 0 from the beginning? does it set it to 0 automatically?

Reply:
It is because I want to have the add question selected if there is no questions when loading the quiz. Without it, I got some problems like having a "movement effect" where the new question button was blinking or because I wasn't able to detect that it is -1 because the user clicked on the add question or because there is no questions...

Reply:
Wouldn't adding some skeleton/loading screen help with the blinking? Like waiting for getting all settings before showing anything? Currently it sounds like a hack otherwise, or I didn't understand the problem?

Reply:
Yeah you're right, I have updated the code...